2021-08-17 14:52:40 [http-nio-8080-exec-10] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 14:54:30 [http-nio-8080-exec-1] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 14:57:21 [http-nio-8080-exec-1] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 15:01:54 [http-nio-8080-exec-3] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 15:02:09 [http-nio-8080-exec-5] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 15:05:32 [http-nio-8080-exec-6] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 15:07:39 [http-nio-8080-exec-8] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 15:08:32 [http-nio-8080-exec-9] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 15:10:40 [http-nio-8080-exec-8] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 15:10:46 [http-nio-8080-exec-10] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Unknown return value type: java.lang.Boolean
2021-08-17 15:16:41 [http-nio-8080-exec-9] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：Optional int parameter 'limit' is present but cannot be translated into a null value due to being declared as a primitive type. Consider declaring it as object wrapper for the corresponding primitive type.
2021-08-17 15:26:43 [http-nio-8080-exec-6] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：null
2021-08-17 15:28:45 [http-nio-8080-exec-1] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException: null
2021-08-17 15:29:45 [http-nio-8080-exec-2] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException: null
2021-08-17 15:29:46 [http-nio-8080-exec-3] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException: null
2021-08-17 15:30:05 [http-nio-8080-exec-2] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 15:30:23 [http-nio-8080-exec-1] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.IllegalStateException: Optional int parameter 'limit' is present but cannot be translated into a null value due to being declared as a primitive type. Consider declaring it as object wrapper for the corresponding primitive type.：Optional int parameter 'limit' is present but cannot be translated into a null value due to being declared as a primitive type. Consider declaring it as object wrapper for the corresponding primitive type.
2021-08-17 15:30:33 [http-nio-8080-exec-3] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 15:31:14 [http-nio-8080-exec-4] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 15:31:46 [http-nio-8080-exec-5] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 15:31:47 [http-nio-8080-exec-6] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 15:31:48 [http-nio-8080-exec-7] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 15:32:02 [http-nio-8080-exec-8] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 15:41:42 [http-nio-8080-exec-2] ERROR o.a.c.c.C.[.[localhost].[/].[dispatcherServlet] - Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is java.lang.NullPointerException] with root cause
java.lang.NullPointerException: null
	at com.xxbb.springbootapi.controller.ApiController.List(ApiController.java:41)
	at com.xxbb.springbootapi.controller.UserController.List(UserController.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:197)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:141)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:106)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:808)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1064)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:681)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:228)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:119)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:190)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:163)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.__invoke(StandardContextValve.java:97)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:41002)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:357)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:382)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:893)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1723)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
2021-08-17 16:02:00 [http-nio-8080-exec-8] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：org.springframework.validation.BindException: org.springframework.validation.BeanPropertyBindingResult: 1 errors
Field error in object 'user' on field 'createTime': rejected value [2020-12-02]; codes [typeMismatch.user.createTime,typeMismatch.createTime,typeMismatch.java.util.Date,typeMismatch]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [user.createTime,createTime]; arguments []; default message [createTime]]; default message [Failed to convert property value of type 'java.lang.String' to required type 'java.util.Date' for property 'createTime'; nested exception is org.springframework.core.convert.ConversionFailedException: Failed to convert from type [java.lang.String] to type [java.util.Date] for value '2020-12-02'; nested exception is java.lang.IllegalArgumentException]：org.springframework.validation.BeanPropertyBindingResult: 1 errors
Field error in object 'user' on field 'createTime': rejected value [2020-12-02]; codes [typeMismatch.user.createTime,typeMismatch.createTime,typeMismatch.java.util.Date,typeMismatch]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [user.createTime,createTime]; arguments []; default message [createTime]]; default message [Failed to convert property value of type 'java.lang.String' to required type 'java.util.Date' for property 'createTime'; nested exception is org.springframework.core.convert.ConversionFailedException: Failed to convert from type [java.lang.String] to type [java.util.Date] for value '2020-12-02'; nested exception is java.lang.IllegalArgumentException]
2021-08-17 16:05:54 [http-nio-8080-exec-1] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 16:06:07 [http-nio-8080-exec-2] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 16:08:03 [http-nio-8080-exec-1] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：java.lang.NullPointerException：null
2021-08-17 16:17:16 [http-nio-8080-exec-4] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.builder.BuilderException: Error invoking SqlProvider method (com.xxbb.springbootapi.helper.UserSqlProvider.findById).  Cause: java.lang.reflect.InvocationTargetException：nested exception is org.apache.ibatis.builder.BuilderException: Error invoking SqlProvider method (com.xxbb.springbootapi.helper.UserSqlProvider.findById).  Cause: java.lang.reflect.InvocationTargetException
2021-08-17 16:27:54 [http-nio-8080-exec-1] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'create_time' in 'field list'
### The error may exist in com/xxbb/springbootapi/mapper/UserMapper.java (best guess)
### The error may involve com.xxbb.springbootapi.mapper.UserMapper.listEntity-Inline
### The error occurred while setting parameters
### SQL: SELECT     `id`, `create_time`, `update_time`, `is_deleted`, `name`, `age`, `sex` FROM  user
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'create_time' in 'field list'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'create_time' in 'field list'：
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'create_time' in 'field list'
### The error may exist in com/xxbb/springbootapi/mapper/UserMapper.java (best guess)
### The error may involve com.xxbb.springbootapi.mapper.UserMapper.listEntity-Inline
### The error occurred while setting parameters
### SQL: SELECT     `id`, `create_time`, `update_time`, `is_deleted`, `name`, `age`, `sex` FROM  user
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'create_time' in 'field list'
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'create_time' in 'field list'
2021-08-17 16:33:48 [http-nio-8080-exec-1] ERROR c.x.springbootapi.interceptor.MyExceptionHandler - 发生异常！信息：org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.builder.BuilderException: Error invoking SqlProvider method (com.xxbb.springbootapi.helper.UserSqlProvider.findById).  Cause: java.lang.reflect.InvocationTargetException：nested exception is org.apache.ibatis.builder.BuilderException: Error invoking SqlProvider method (com.xxbb.springbootapi.helper.UserSqlProvider.findById).  Cause: java.lang.reflect.InvocationTargetException
